deployment/armada/values.yaml

```diff
@@ -1,34 +1,126 @@
+nameOverride: ""
+fullnameOverride: ""
+
 image:
+  # image repository
   repository: gresearchdev/armada-server
-  tag: LATEST
+  # image tag
+  tag: v0.3.39
```

Comment:
Usa a variable? All components should normally be set to the same version.

Reply:
What did we agree for the tag?
Should we keep LATEST here and in the Chart YAML like it was before, with the note that LATEST isn't a valid tag for Helm charts and the Chart is not well-formed?

When we release a versioned chart, we do version substitution there

Reply:
Could we have the tag in the helm files be a variable and then set the values in the values.yaml to the latest tag when we release the charts?

Reply:
Helm doesn't allow templating in the Chart.yaml sadly